app/design/frontend/AndriiShkrebtii/luma/Magento_Catalog/templates/product/view/message.phtml

```diff
@@ -0,0 +1,6 @@
+<?php /** @var \Magento\Framework\View\Element\Template $block */ ?>
+<?php /** @var \Magento\Framework\Escaper $escaper */ ?>
+
+<div class="message-warning warning message">
+    <p><?= $escaper->escapeHtml(__($block->GetData('message'))) ?></p>
```

Comment:
@jimmywilson111 This MUST be `getData`, but not `GetData` (first letter is lowercase)